sw/airborne/peripherals/mpu9250.c

```diff
@@ -110,6 +114,27 @@ void mpu9250_send_config(Mpu9250ConfigSet mpu_set, void *mpu, struct Mpu9250Conf
       mpu_set(mpu, MPU9250_REG_INT_ENABLE, (config->drdy_int_enable << 0));
       config->init_status++;
       break;
+
```

Comment:
Sorry,  Is your means move all follow to mpu9250_configure_i2c_slaves?  The follow code is executed after mpu9250_configure_i2c_slaves?